### PR TITLE
Adds lost lane to 404 page

### PR DESCRIPTION
**Pull request by oliviawongnyc**
[OE-851](https://jira.nypl.org/browse/OE-851): Adds `Are you Lost?` lane to `404` page

### CODE REVIEW

**Review by samanthaandrews — APPROVED** [review]
LGTM

File: apps/oew/src/components/chakra-ui/fetching.ts [hunk 1]
```diff
@@ -59,6 +59,16 @@ export const fetchCuratedCollections = async () => {
...
 };
 
+/**
+ * Fetches the lane data for the 404 page.
+ * This is a list of books about being lost.
+ */
+export const fetchLostCollection = async () => {
+  const lostBooksUrl = makeCMUrl(lostCollectionEndpoint);
+
+  return await Promise.resolve(fetchCollectionBooks(lostBooksUrl));
```
Comment (kristojorg) [inline apps/oew/src/components/chakra-ui/fetching.ts:69]: Just curious about this use of Promise.resolve. I think you should be able to return `fetchCollectionBooks(lostBooksUrl)` without that

**Review by kristojorg — APPROVED** [review]
Looks good! Just the one little comment

File: apps/oew/src/components/chakra-ui/fetching.ts (hunk 1 above)
Comment (oliviawongnyc) [reply apps/oew/src/components/chakra-ui/fetching.ts:69]: Oops missed this in the flurry before my vacation. I removed and it still seems to work. I had originally added to mimic the fetchCuratedCollections function above.

Comment (kristojorg) [reply apps/oew/src/components/chakra-ui/fetching.ts]: Ahh I see I see. In that case, `Promise.all` is used to create one promise that waits for all of the individual promises passed in to it (and `fetchCollectionBooks` returns one)

**Comment by samanthaandrews** [comment]
If this has been approved by QA, let's merge!

**Pull request merged**